### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by PaulREnglish**
Added project section, contact me section.  Reworked the Navbar.  This should be it's finished state.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <head>
 	
 	<title>	
-		Paul Robert English
+		Paul English
 	</title>
 	
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (PaulREnglish) [inline index.html:10]: needs closing characters />, also is initial scale supposed to be in the quotes? some help would be appreciated. thanks

Comment (xz-dice) [reply index.html:10]: According to t'internet, the meta tag is okay to be closed just with a ">" as HTML5 understands what this means. It also seems that the whole string for the content attribute is "width=device-width, initial-scale=1.0", so the initial scale should stay within the quotation marks.

File: index.html [hunk 2]
```diff
@@ -96,13 +69,106 @@ <h2> About me: </h2>
 		</div>
 	
 	
+	<section class="aboutMe">
+		<h2 id="About Me"> About me: </h2>
+		
+		<p>
+			I am an aspiring software developer who finished his MMath from the university of Oxford in the year 2020.  In my spare time I like to make maps for the doom and hexen games and apply my knowledge of ACS and DECORATE to make modifications to the enemies and levels.  (If you want to see some of the maps I've made, take a look at the <a href="https://doomwiki.org/wiki/Zedonk">doom wiki.</a>)  I've also made a Unity game.
```
Comment (xz-dice) [inline index.html]: Wrap this paragraph so we can read the whole thing, makes it easier to read the code.

File: styles.css [hunk 3]
```diff
@@ -7,13 +7,14 @@ html{
 }
 
 /* centers the webpage */
-body{
+/*body{
 	display: flex;
 	justify-content: center;
-}
+}*/
```
Comment (xz-dice) [inline styles.css]: If this is commented out can this be removed?

File: styles.css [hunk 4]
```diff
@@ -46,16 +49,25 @@ nav > div{
 	border: 2px solid #777674;
 }
 
-.navSmallScreen{
-	display: none;
```
Comment (xz-dice) [inline styles.css:50]: does this element still need a class if it's not referenced directly in the css?

File: styles.css [hunk 5]
```diff
@@ -46,16 +49,25 @@ nav > div{
...
+.navigation > a{
+	text-decoration: none;
+}
+
+.navigation > a:visited{
+	color: #000000;
+}
+
+.navigation > a:link{
+	color: #000000;
```
Comment (xz-dice) [inline styles.css]: This didn't change anything in the inspector when I turned it off (or when I clicked the links)

File: styles.css [hunk 6]
```diff
@@ -46,16 +49,25 @@ nav > div{
...
 
-/* title and about me section*/
+/* title section*/
 header{
 	display: flex;
-	/*height: 800px;*/
+	height: 700px;
 	justify-content: center;
 	align-items: center;
 	padding: 180px 0;
```
Comment (xz-dice) [inline styles.css:58]: Turning padding off in the inspector doesn't change anything in my browser. This is the same for the 'background-repeat' declaration. I think that when you use background-size: cover in this instance, it means you don't have to use 'no-repeat'.

**Review by xz-dice — CHANGES_REQUESTED** [review]
It's worth going through and removing extra space in the HTML and CSS - particularly where elements have been nested - as this makes it easier to read the code. If there is any commented out code in the CSS that is unnecessary, delete this too. Otherwise, really clean straightforward code - great job!

File: index.html (hunk 1 above)
Comment (PaulREnglish) [reply index.html:10]: awesome thanks!

File: styles.css (hunk 5 above)
Comment (PaulREnglish) [reply styles.css]: Strange it doesn't come up on my inspector.  But turning off manually in the code definitely has an effect in my browser.

File: styles.css (hunk 3 above)
Comment (PaulREnglish) [reply styles.css]: yh I'll remove it

File: index.html (hunk 2 above)
Comment (PaulREnglish) [reply index.html]: done!

File: styles.css (hunk 6 above)
Comment (PaulREnglish) [reply styles.css:58]: Thanks!
No repeat has been removed but changing the height has an affect in browser.  I think it only has an affect if you're screen is very wide like mine.

File: styles.css (hunk 4 above)
Comment (PaulREnglish) [reply styles.css:50]: I think the minus signs mean I've deleted that piece of code

**Pull request merged**